### PR TITLE
Build/python typing

### PR DESCRIPTION
**Pull request by juskek**

### CODE REVIEW

File: sdk-python/omnilogger/get_pricing.py [hunk 1]
```diff
@@ -38,8 +39,8 @@ def load_llm_pricing():
     return pricing
 
 
-def read_json_at(path: str) -> dict:
-    with open(path, "r") as json_file:
+def read_json_at(path: str) -> dict[str, Any]:
```
Comment (CallumHemsley) [inline sdk-python/omnilogger/get_pricing.py:42]: aren't all dicts `dict[str, x]`? 🤔 why was this change necessary

**Review by CallumHemsley — APPROVED** [review]

File: sdk-python/omnilogger/get_pricing.py (hunk 1 above)
Comment (juskek) [reply sdk-python/omnilogger/get_pricing.py:42]:
```
dict --> dict[Unknown, Unknown]

you can have 
dict[str,Any]
dict[Any,str]

```
